### ISSUE TITLE
Client won't get static IP-Adresse

### ISSUE THREAD

**Issue by white-rabbit07** [coova/coova-chilli#223]
Hi.
I use coova-chilli   1.3.0 on Ubuntu 12.04.5.
I want to use static IPs for some clients and set this in etc/chilli/config:
`HS_STATIP=172.20.8.0
HS_STATIP_MASK=255.255.255.224 
HS_DYNIP=172.20.0.0
HS_DYNIP_MASK=255.255.248.0   
`
When I use "chilli_query listippool" I get this:
`
-- Dynamic Pool
------------------------------------------------------------
Unit   0 :  -1/  1 :      172.20.0.2 :
Unit   1 :   0/  2 :      172.20.0.3 :

[...]

and later:

-- Static Pool
------------------------------------------------------------
Unit 2045 :  -1/2046 :         0.0.0.0 : static
Unit 2046 : 2045/2047 :         0.0.0.0 : static
`
Why does 0.0.0.0 and not my static IP range appear there?
Thanks for help!
When a client with Mac-Auth. requests an IP-Adress that *SHOULD* be static it gets a dynamic adress... I can't explain that as it worked some months ago :(

**Comment by nzamps** [comment]
When a static ip is in-use it is shown like:

-- Static Pool     ------------------------------------------------------------
Unit  70 : -inuse- :     192.168.1.56 : static mac=00-0C-01-01-5F-7B ip=192.168.1.56 age=3

**Comment by white-rabbit07** [comment]
No, I can't confirm that. My static IPs are not listet there :(
